octoprint_printoid/pisupport_notifications.py

```diff
@@ -0,0 +1,58 @@
+from .alerts import Alerts
+
+
+class piSupportNotifications:
```

Comment:
This class could be named `PiAlertNotifications` for consistency

Reply:
Check - will look asap